docs/root/intro/arch_overview/http/http3.rst

```diff
@@ -0,0 +1,40 @@
+.. _arch_overview_http3:
+
```

Comment:
probably we want a page title above the upstream/downstream subtitles - something like "HTTP3 overview"